packages/InteractiveProfilingTool-Core.package/IPTMessageTallyWrapper.class/instance/getRelativeTime.st

```diff
@@ -0,0 +1,6 @@
+information extraction
+getRelativeTime
```

Comment:
Gibt es einen Grund dafür, dass die eigentlich Berechnung in die Methode `calculateRelativeTime` ausgelagert ist? Mich hat das auf den ersten Blick verwirrt, weil unklar ist, welche von beiden Methoden die öffentlich verwendbare ist ([und ich bin damit offensichtlich nicht alleine](https://github.com/hpi-swa-teaching/InteractiveProfilingTool/pull/36/files#r294283296)). Außerdem haben wir dadurch gerade duplizierte Tests.

Die Methode könnte vollständig in wenigen Zeilen geschrieben werden.

```smalltalk
"All calls in the tree have a time of 0. Catch this case to avoid ZeroDivide Errors later on."
self totalTimeOfOverallTree isZero ifTrue: [^ 100].

^ self getAbsoluteTime / self totalTimeOfOverallTree * 100
```